kalite/distributed/static/js/distributed/videoplayer.js

```diff
@@ -1,504 +0,0 @@
-    window.VideoPlayerState = {
```

Comment:
Yeah - but now you can just blame me. I did also move a bunch of stuff around, moving a lot of what was in the model into the view.